### PR TITLE
feat(cdu): check TO data

### PR DESCRIPTION
**Pull request by MisterChocker**
## Summary of Changes

- [ ] #3082 is implemented
- [x] Trigger check take to data after changing departure runway or relevant fields on takeoff perf page
- [x] Add individual and dynamic check for confirmation
- [x] Add check to data message

## References

image
image

## Additional context

Discord username (if different from GitHub): DerL30N#3751

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Comment by beheh** [comment]
Just FYI, as I'm away from my sim next week I'm trying really hard to get #1824 done this week, my work permitting.

**Comment by MisterChocker** [comment]
> Just FYI, as I'm away from my sim next week I'm trying really hard to get #1824 done this week, my work permitting.

Thanks for the heads up, take your time.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js [hunk 1]
```diff
@@ -160,9 +185,25 @@ class CDUPerformancePage {
...
             }
         };
+        let next = "NEXT";
+        let nextPage = "PHASE>";
+        mcdu.onRightInput[5] = () => {
+            CDUPerformancePage.ShowCLBPage(mcdu);
+        };
+        if (!mcdu._v1Checked || !mcdu._vrChecked || !mcdu._v2Checked) {
+            next = "CONFIRM";
+            nextPage = "TO DATA{small}*{end}";
```
Comment (MisterChocker) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js:199]:
```suggestion
            nextPage = "TO DATA*";
```

**Comment by MisterChocker** [comment]
Wasn't able to use this branch cause of some git hub shenanigans #3285

**Pull request closed without merging**